### DISCUSSION TITLE
Visual Studio GUI for managing vcpkg dependencies?

### DISCUSSION THREAD

**Discussion by AugP** [microsoft/vcpkg#35404] [Ideas]
I've seen this topic come up a few times in feedback, so I wanted to make a thread on it. Are you interested in a GUI for managing vcpkg dependencies in Visual Studio? If so, please upvote this post so we can track it. 

Also, feel free to comment below about what features you are looking in such a GUI.

**Comment by Osyotr** [comment]
For inspiration:
https://vcpkg.link/
https://www.jetbrains.com/help/clion/package-management.html

**Comment by pjmlp** [comment]
Definitely, it would be great to be at least on the same level as NuGET VS experience, and out of the box without having to call vcpkg from the command line, as it is still the case after the integration on VS installer.

**Comment by TheBarrow** [comment]
This would be especially nice, since NuGet integration is pretty buggy for shared projects. We regularly have to do weird workarounds to get Intellisense to recognize NuGet packages, and sometimes we have to completely reinstall to get them to build (MSVC 17.12.4)
